models/test_fixtures.go

```diff
@@ -35,5 +35,37 @@ func LoadFixtures() error {
 	if err != nil {
 		fmt.Printf("LoadFixtures failed after retries: %v\n", err)
 	}
+	// Now if we're running postgres we need to tell it to update the sequences
+	if x.Dialect().DriverName() == "postgres" {
+		results, err := x.QueryString(`SELECT 'SELECT SETVAL(' ||
```

Comment:
Building the statement in SQL gives me some creeps... I'd rather build the statement in go, but I guess it's not much of a change since there are literals involved that can't be passed as parameters anyway (and hey! this is the test instance).